src/xpub/entity.rs

```diff
@@ -81,55 +84,52 @@ impl AccountXPub {
 }
 
 #[derive(Builder, Clone, Debug)]
-pub struct NewAccountXPub {
-    pub(super) db_uuid: uuid::Uuid,
+pub struct NewXpub {
+    pub(super) id: XpubId,
     pub(super) account_id: AccountId,
     #[builder(setter(into))]
-    pub(super) key_name: String,
+    pub(super) name: String,
     pub(super) original: String,
     pub(super) value: XPubValue,
+    pub(super) fingerprint: XPubId,
```

Comment:
I think i explained that we should use accessor instead of adding a new field here.